sentry_member_team_map_withoutSCIM.py

```diff
@@ -130,7 +144,14 @@ def update_team_reg(self, memberid, teamname):
             }
             data["userName"] = member.get('email')
             data['email'] = member.get('email')
+
+            ### NOTE, Nov 2022: I don't think it's possible to create members via API anymore, the API must have changed.
+            #   This portion of the script should be rewritten to reflect it.
+            #   It's best to invite these members manually. If there are a lot of members, Sentry customer should coordinate
+            #   with their Sentry contacts to figure out the best approach.
```

Comment:
We should figure out what behavior we want to update the script with, given that this functionality no longer seems to work (I discovered this while running the script for a customer). Right now this will just fail silently I believe.

Reply:
⚠️ - I think whoever reviews this should ensure we come to a team decision about this before merging this PR.